src/components/common/Extensions/UserHub/partials/CryptoToFiatTab/partials/ReceiveCard.tsx

```diff
@@ -32,25 +44,38 @@ const ReceiveCard: FC<ReceiveCardProps> = ({
   isFormDisabled,
   handleSetMax,
 }) => {
-  const name = 'convertedAmount';
-
-  const { getValues, setValue, getFieldState } = useFormContext();
+  const name = TransferFields.CONVERTED_AMOUNT;
 
-  const { error } = getFieldState(name);
+  const {
+    setValue,
+    formState: { errors },
+  } = useFormContext();
+  const amountValue = useWatch({ name });
+  const error = get(errors, name)?.message as Message | undefined;
 
-  const amountValue = getValues(name);
+  const { currency: selectedCurrency } = useCurrencyContext();
```

Comment:
The selected currency needs to come from the details the user has entered in the crypto to fiat user profile section rather than the currency context

Reply:
Hey @iamsamgibbs I already did this change into this PR https://github.com/JoinColony/colonyCDapp/pull/2988 as I realised it wasn't properly set and shouldn't have used the useCurrencyContext()